### PR TITLE
Clean up logging

### PR DESCRIPTION
**Pull request by qwerty287**
- use `Err` method instead of format strings
- use `Msg` if no format string is used

### CODE REVIEW

File: server/queue/fifo.go [hunk 1]
```diff
@@ -241,17 +240,6 @@ func (q *fifo) process() {
...
-		// the risk of panic is low. This code can probably be removed
-		// once the code has been used in real world installs without issue.
-		if err := recover(); err != nil {
-			const size = 64 << 10
-			buf := make([]byte, size)
-			buf = buf[:runtime.Stack(buf, false)]
-			log.Error().Msgf("queue: unexpected panic: %v\n%s", err, buf)
-		}
-	}()
-
```
[the remark this answers is not in the document]
Comment (6543) [reply server/queue/fifo.go:254]: please no remove it!!!

last time i did so, woodpecker did not work anymore.

if you want to remove it, do it in a dedicated pull so we can revert it easyly

**Review by 6543 — CHANGES_REQUESTED** [review]

File: server/queue/fifo.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (6543) [reply server/queue/fifo.go:254]: no just exclude this change from this pull and add it to it's own

**Comment by xoxys** [comment]
Can we use constituent upper/lowercases for log messages? Some log messages start with upper, some with lowercase.

**Review by anbraten — APPROVED** [review]

**Review by 6543 — APPROVED** [review]

**Comment by xoxys** [comment]
Ok 3 people agreed to normalize upper/lower cases, but the PR was merged without fixing it :smile: Should I create an issue to track it?

**Comment by 6543** [comment]
the "linter explosin" pull do have a linter for that if i remember, ... so this part just has to be picket out and it should lint & you also have the fixes there

**Pull request merged**